var/spack/repos/builtin/packages/ninja/package.py

```diff
@@ -14,8 +14,10 @@ class Ninja(Package):
 
     homepage = "https://ninja-build.org/"
     url      = "https://github.com/ninja-build/ninja/archive/v1.7.2.tar.gz"
+    git      = "https://github.com/Kitware/ninja.git"
 
-    version('1.8.2', '5fdb04461cc7f5d02536b3bfc0300166')
+    version('kitware', branch='features-for-fortran')
+    version('1.8.2', '5fdb04461cc7f5d02536b3bfc0300166', preferred=True)
```

Comment:
In Spack, any named version that is not `develop` is automatically *older* than the oldest known numbered version of a package.

(Also, `develop` is not `preferred` but the latest numbered version is.)

Therefore, the `preferred=True` can be skipped here.